### ISSUE TITLE
TypeError: sigma.init is not a function

### ISSUE THREAD

**Issue by zhangqteradata** [jacomyal/sigma.js#855]
when updating to the version of 1.2.0,I found the TypeError :sigma.init is not a function.I want to know whether there was  another method

**Comment by Yomguithereal** [comment]
I you are upgrading from a <= 1 version this is to be expected. But I don't recall what was the use of this method.

**Comment by zhangqteradata** [comment]
Thank you,I has tried to sloved the problem by the method that sigma.init()=new sigma() and whether it is OK?

**Comment by Yomguithereal** [comment]
I don't think this will be OK. I you try to migrate from <= 1 to sigma v1 you'll have a lot of code to rewrite.
